packages/main-ui/src/layouts/congraturation_popup.rs

```diff
@@ -1,17 +1,21 @@
 #![allow(non_snake_case)]
 use dioxus::prelude::*;
 use dioxus_popup::PopupService;
+use dioxus_translate::*;
 
 use crate::{components::icons::Logo, theme::Theme};
+use super::i18n::HeaderTranslate;
 
 #[component]
 pub fn CongraturationPopup(
     #[props(default ="congraturation_popup".to_string())] id: String,
     #[props(default ="".to_string())] class: String,
+    lang: Language,
 ) -> Element {
     let theme: Theme = use_context();
     let theme = theme.get_data();
     let mut popup: PopupService = use_context();
+    let tr = translate::<HeaderTranslate>(&lang);
```

Comment:
Refactor to use as below

```suggestion
    let tr = translate::<CongratuationPopupTranslate>(&lang);
```